velox/functions/CoverageUtil.h

```diff
@@ -0,0 +1,316 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include <fstream>
+#include <iostream>
+#include <string>
+#include <vector>
+#include "velox/functions/FunctionRegistry.h"
+
+using namespace facebook::velox;
```

Comment:
Do not use "using" directive in header files. Move the implementation into .cpp file and leave only necessary declarations in the .h file.

Reply:
Thanks for your comments. just fixed it.